src/openforms/forms/static/forms/scss/components/admin/_form-object-tools.scss

```diff
@@ -0,0 +1,13 @@
+.form-object-tools {
+
+  ul {
+    margin: 0;
+  }
+
+  &__loading {
+    a {
+      opacity: 0.65;
+      cursor: not-allowed;
+    }
+  }
+}
```

Comment:
Don't target HTML elements, only target class names. This allows us to change the markup without having to update styling.